ariadne_codegen/settings.py

```diff
@@ -49,7 +53,21 @@ def __post_init__(self):
         if self.schema_path:
             assert_path_exists(self.schema_path)
 
-        self.remote_schema_headers = resolve_headers(self.remote_schema_headers)
```

Comment:
It looks like a crucial line of code was removed, which is necessary for the logic to work correctly. Please restore it, as its absence is causing functionality issues.